recipes/valijson/all/conanfile.py

```diff
@@ -10,7 +10,7 @@ class ValijsonConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/tristanpenman/valijson"
     license = "BSD-2-Clause"
-    no_copy_source = True
+    exports_sources = ["include/*", "patches/**"]
```

Comment:
I did not see anything (in this PR) in the "include"  folder, is there a reason this is added?

Reply:
If I make it `no_copy_source = True` as it was, then I get an error:

```
ERROR: valijson/0.3: Error in build() method, line 27
	tools.patch(**patch)
	FileNotFoundError: [Errno 2] No such file or directory: 'patches/nlohmann_json.patch'
```

I'm not sure how patching header-only libraries works.

Reply:
Ahhh ... it needs to happen in the build step, which is skipped with `no_copy_source = True`! Since you applied the other comment, I suspect this will work.

```suggestion
    exports_sources = ["patches/**"]
```

Reply:
Thanks @prince-chrismc